### DISCUSSION TITLE
Centred tab titles

### DISCUSSION THREAD

**Discussion by ProudlyTrad** [rafaelmardojai/firefox-gnome-theme#373] [Ideas]
One of the things i love about Epiphany is the centred tab titles. Can you bring this feature to the firefox theme?

**Comment by rafaelmardojai** [comment]
What do you mean? Tabs titles are centered.
![Captura de pantalla de 2022-05-16 07-25-30](https://user-images.githubusercontent.com/6210397/168591998-7f4188fd-785d-4aef-8444-181c132103f0.png)
